src/components/dashboard/Receive.js

```diff
@@ -102,4 +80,15 @@ Receive.navigationOptions = {
   title: RECEIVE_TITLE
 }
 
-export default Receive
+const getStylesFromProps = ({ theme }) => {
+  return {
+    marginTop: {
```

Comment:
I don't feel right with this naming. Looks more like a temp name that you forgot to change. Probably something more specific, but different form a CSS property.

On the other hand, what you think about not creating those classes and applying the `theme.paddings.defaultMargin` to the `marginTop/Bottom` style props of the elements?

```javascript
<Section.Text style={{ marginBottom: theme.paddings.defaultMargin }}>
```

Reply:
Since these are different components (buttons and Texts) seems like we might ending up adding lots of properties to several components to do things like this. I would keep the margin for styles for now I think. 
Also, margins shouldn't be very common since we are using flex align for lots of this layout use cases

Do you agree in to keep it as it is for now? I changed the names as suggested though

Reply:
Yes, sure! Leave it as it is. Thanks